tests/unit/messages/formats/html/fixtures/all_icons.html

```
@@ -1 +1 @@
-<div style="font-family:-apple-system,BlinkMacSystemFont,'Segoe UI',sans-serif;font-size:14px;line-height:1.5;color:#1f2937;background-color:#ffffff;border:1px solid #e5e7eb;border-radius:6px;padding:24px 32px 32px 32px;max-width:800px"><div style="margin:0 0 12px"><div style="margin:0;">RED_TRIANGLE <span style="margin-right:4px;">🔺</span> X <span style="margin-right:4px;">❌</span> WARNING <span style="margin-right:4px;">⚠️</span> EXCLAMATION <span style="margin-right:4px;">❗</span> CHECK <span style="margin-right:4px;">✅</span> MAGNIFYING_GLASS <span style="margin-right:4px;">🔎</span> HAMMER_AND_WRENCH <span style="margin-right:4px;">🛠️</span> POLICE_LIGHT <span style="margin-right:4px;">🚨</span> INFO <span style="margin-right:4px;">ℹ️</span> EYE <span style="margin-right:4px;">👁️</span> GEAR <span style="margin-right:4px;">⚙️</span> BELL <span style="margin-right:4px;">🔔</span> GEM <span style="margin-right:4px;">💎</span> SPARKLES <span style="margin-right:4px;">✨</span> LINK <span style="margin-right:4px;">🔗</span></div></div></div>
+<div style="font-family:-apple-system,BlinkMacSystemFont,'Segoe UI',sans-serif;font-size:14px;line-height:1.5;color:#1f2937;background-color:#ffffff;border:1px solid #e5e7eb;border-radius:6px;padding:24px 32px 32px 32px;max-width:800px"><div style="margin:0 0 12px"><div style="margin:0;">RED_TRIANGLE <span style="margin-right:4px;">🔺</span> X <span style="margin-right:4px;">❌</span> WARNING <span style="margin-right:4px;">⚠️</span> EXCLAMATION <span style="margin-right:4px;">❗</span> CHECK <span style="margin-right:4px;">✅</span> MAGNIFYING_GLASS <span style="margin-right:4px;">🔎</span> HAMMER_AND_WRENCH <span style="margin-right:4px;">🛠️</span> POLICE_LIGHT <span style="margin-right:4px;">🚨</span> INFO <span style="margin-right:4px;">ℹ️</span> EYE <span style="margin-right:4px;">👁️</span> GEAR <span style="margin-right:4px;">⚙️</span> BELL <span style="margin-right:4px;">🔔</span> GEM <span style="margin-right:4px;">💎</span> SPARKLES <span style="margin-right:4px;">✨</span> LINK <span style="margin-right:4px;">🔗</span> NEW <span style="margin-right:4px;">🆕</span></div></div></div>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)